### PR TITLE
docs: Update analytics events update and new analytic events templates

### PR DESCRIPTION
**Pull request by ainfeld**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/analytics-events-update.md [hunk 1]
```diff
@@ -1,29 +1,31 @@
...
 ### Updates needed
-
 `<Event name>`
-
 - Current: `<Description of how the event works currently>`
 - Requested change: `<Description of the update needed>`
 
-#### People to cc
+Checklist 
+- [ ] Once event updates are included in Staging ping @ainfeld to check analytics
```
Comment (metakuni) [inline .github/ISSUE_TEMPLATE/analytics-events-update.md:22]:
```suggestion
- [ ] Once event updates are deployed to Staging ping @ainfeld to check analytics 
```

File: .github/ISSUE_TEMPLATE/analytics-events-update.md [hunk 2]
```diff
@@ -1,29 +1,31 @@
...
 `<Event name>`
-
 - Current: `<Description of how the event works currently>`
 - Requested change: `<Description of the update needed>`
 
-#### People to cc
+Checklist 
+- [ ] Once event updates are included in Staging ping @ainfeld to check analytics 
+- [ ] Verify events are showing up in the Staging space in Plausible correctly 
+- [ ] Once event updates are included in Prod ping @ainfeld
```
Comment (metakuni) [inline .github/ISSUE_TEMPLATE/analytics-events-update.md:24]:
```suggestion
- [ ] Once event updates are deployed to Prod ping @ainfeld 
```

File: .github/ISSUE_TEMPLATE/new-analytic-events.md [hunk 3]
```diff
@@ -1,28 +1,33 @@
...
-
+**Expectations**: `<usage of this feature or functionality>` 
+
+### New events to add 
+`<suggested event name>`: `<information that should be stored in the payload (if applicable)>`
+- `<Description of what causes the event to fire>`
+
+Checklist 
+- [ ] Add events to the custom goals section of both the staging and prod Plausible spaces 
+- [ ] If rdev environment is created for this new feature, once it is available ping @ainfeld to check analytics
```
Comment (metakuni) [inline .github/ISSUE_TEMPLATE/new-analytic-events.md:23]:
```suggestion
- [ ] If rdev environment is created for this new feature, once it has been deployed, ping @ainfeld to check analytics 
```

File: .github/ISSUE_TEMPLATE/new-analytic-events.md [hunk 4]
```diff
@@ -1,28 +1,33 @@
...
+**Expectations**: `<usage of this feature or functionality>` 
+
+### New events to add 
+`<suggested event name>`: `<information that should be stored in the payload (if applicable)>`
+- `<Description of what causes the event to fire>`
+
+Checklist 
+- [ ] Add events to the custom goals section of both the staging and prod Plausible spaces 
+- [ ] If rdev environment is created for this new feature, once it is available ping @ainfeld to check analytics 
+- [ ] Once feature launches in Staging ping @ainfeld to check analytics again
```
Comment (metakuni) [inline .github/ISSUE_TEMPLATE/new-analytic-events.md:24]:
```suggestion
- [ ] Once feature is deployed to Staging ping @ainfeld to check analytics again  
```

File: .github/ISSUE_TEMPLATE/new-analytic-events.md [hunk 5]
```diff
@@ -1,28 +1,33 @@
...
+### New events to add 
+`<suggested event name>`: `<information that should be stored in the payload (if applicable)>`
+- `<Description of what causes the event to fire>`
+
+Checklist 
+- [ ] Add events to the custom goals section of both the staging and prod Plausible spaces 
+- [ ] If rdev environment is created for this new feature, once it is available ping @ainfeld to check analytics 
+- [ ] Once feature launches in Staging ping @ainfeld to check analytics again  
+- [ ] Verify events are showing up in the Staging space in Plausible correctly 
+- [ ] Once feature launches to Prod ping @ainfeld
```
Comment (metakuni) [inline .github/ISSUE_TEMPLATE/new-analytic-events.md:26]:
```suggestion
- [ ] Once feature is deployed to Prod ping @ainfeld 
```

**Review by metakuni — APPROVED** [review]
Added a few nit wording edits, but feel free to ignore!

**Review by tihuan — APPROVED** [review]
LGTM too! Kuni's suggestions LGTM as well :+1: Thanks so much for updating this, @ainfeld !

I see that the formatting test failed, I can help run prettier tomorrow morning to fix that!

**Comment by ainfeld** [comment]
Thanks everyone! I love all of the suggestions and have accepted them.

**Comment by tihuan** [comment]
Just ran `pre-commit run --all-files` to fix formatting and enabled automerge. Thanks again, @ainfeld 🎉

**Pull request merged**